pelutils/ds/plots.py

```diff
@@ -146,3 +139,84 @@ def get_bins(
     if ignore_zeros:
         x, y = x[y>0], y[y>0]
     return x, y
+
+class Figure:
+    """ Used for more ergonomic plotting. Simple usecase:
+    ```py
+    with Figure("figure.png", figsize=(20, 10), fontsize=50):
+        plt.plot(x, y)
+        plt.title("Very large title")
+        plt.grid()
+    # The finished figure is saved to "figure.png"
+    # All settings are reset here
+    ```
+    It is also possible to access the figure and axis produced by plt.subplots.
+    These have type hinting, so it is for once possible to known what methods
+    and attributes exist on fig and ax.
+    In the example, the seaborn style is used (similar to `plt.style.use("seaborn"))`.
+    ```py
+    with Figure(show=True, figsize=(20, 10), stylesheet="seaborn") as f:
+        f.ax.set_title("Normal sized title")
+        f.figure.add_axes(...)
+    # No path has been specified, so the figure is not saved, but a window is shown.
+    ``` """
+
+    def __init__(
+        self,
+        savepath:     Optional[str] = None, *,
+        nrow:         int  = 1,
+        ncol:         int  = 1,
+        tight_layout: bool = True,
+        show:         bool = False,
+        stylesheet:   Optional[str] = None,
+        # Arguments below here go into mpl.rcParams
```

Comment:
Er det her ikke lidt upraktisk? Du kan jo ikke rigtig bede om den globale rc-param-opførsel? Burde man ikke kunne sætte None på de der grafiske parametre og så opdaterer den ikke rc-param? Og burde de default'e til None? Kan ikke lige gennemskue den bedste løsning

Reply:
Jo, det er ikke helt optimalt - men jeg kan ikke se formålet med at bede om den globale rc-param. Jeg har lige set mig rundt på interwebs, og der er åbenbart en `with plt.rc_context`: https://www.tutorialspoint.com/how-to-set-local-rcparams-or-rcparams-for-one-figure-in-matplotlib, som jeg tænker kan bruges til at gøre det lidt pænere uden at pille ved de globale indstillinger. Jeg har overvejet at defaulte til None, men som jeg ser det er der to problemer:
- Det er ikke ligetil at se, hvad deres standardværdi er
- Hvad skulle de så defaulte til? Standard rcParams eller vores egne, hjemmebryggede standardværdier? IMO er standardværdierne ret dårlige

Reply:
@sorenmulli Jeg har skubbet et nyt bud på `Figure-klassen`. Tanker?